kornia/enhance/add_snow.py

```diff
@@ -0,0 +1,23 @@
+import torch
+from kornia.color import rgb_to_hls, hls_to_rgb
+from kornia.testing import KORNIA_CHECK_IS_COLOR, KORNIA_CHECK_IS_TENSOR
+
+
+def add_snow(image: torch.Tensor, snow_coef: torch.Tensor, brightness_coef: torch.Tensor) -> torch.Tensor:
+    """Add snow to the image.
+
+    Snow is added in the form of bleach of some pixels.
+    """
+
+    KORNIA_CHECK_IS_TENSOR(image)
+    KORNIA_CHECK_IS_COLOR(image, f"with shape {image.shape}")
+
+    snow_coef *= 0.5  # = 255 / 2
+    snow_coef += 0.33  # = 255 / 3
```

Comment:
i don't understand this coefficients. Can you add some comments ?

Reply:
I forgot to remove the comments, but the idea of changing snow coefficients came from the original implementation where they have a 0-255 range for RGB. I wanted to discuss whether this change is needed. It can be useful if the randomly determined snow coefficient is very small, and whether we always want to apply the snow.

Reply:
got it but i guess that this coefficient should represent some kind of magnitude right ?